docs/reference/release-policy.md

```diff
@@ -3,72 +3,113 @@ id: release-policy
 
 :::
 
-## Provisioning in SaaS
+## SaaS provisioning
 
-In our managed service, we differentiate between components that are part of a Camunda 8 cluster and components that are outside of the cluster.
+In Camunda 8 SaaS we differentiate between components that are part of a Camunda 8 cluster (cluster components), and components outside the cluster (non-cluster components).
 
-A cluster usually consists of:
+### Cluster components
 
-- Zeebe
-- Operate
-- Tasklist
-- Optimize
+A cluster typically consists of the following components:
 
-For components **outside of the cluster**, we release new versions continuously and update customers to the latest version automatically whenever it is ready to be shipped.
+- [Zeebe](/docs/components/zeebe/zeebe-overview.md)
+- [Operate](/docs/components/operate/operate-introduction.md)
+- [Tasklist](/docs/components/tasklist/introduction-to-tasklist.md)
+- [Optimize]($optimize$/components/what-is-optimize)
 
-These components include:
+You can provision cluster components using one of two channels, following the [Camunda release policy](https://camunda.com/release-policy/).
 
-- Modeler (Web)
-- Connectors
-- Console
+![Stable and alpha channels when provisioning a cluster](./img/channels.png)
 
-Admins can [enable alpha features](/components/console/manage-organization/enable-alpha-features.md) for components outside of the cluster in the organization settings screen.
+#### Stable channel
 
-For components inside a **cluster**, Camunda provides two channels for provisioning and follows the [Camunda release policy](https://camunda.com/release-policy/):
+You can use the **Stable** channel to access [general availability](/docs/reference/alpha-features.md#general-availability-ga) features for cluster components.
 
-- **Stable**: General availability features for cluster components are available through the stable channel. This channel provides the latest feature and patch releases ready for most users at minimal risk. The releases follow semantic versioning and can be updated to the next minor or patch release without data loss.
-- **Alpha**: Alpha features for cluster components are available through the alpha channel. This channel provides preview releases in preparation for the next stable release. They provide a short-term stability point to test new features and give feedback before they are released to the stable channel. Try these to ensure the upcoming release works with your infrastructure. These releases cannot be updated to a newer release, and therefore are not meant to be used in production.
+- Provides the latest feature and patch releases ready for most users at minimal risk.
+- Releases follow semantic versioning and can be updated to the next minor or patch release without data loss.
+- On the stable channel, the last three supported minor versions are made available for provisioning.
```

Comment:
I think the reality is that we provide all supported minor versions (ie. see end of [maintenance](https://docs.camunda.io/docs/next/reference/release-notes/) => this can be 3, or 4 depending on how many releases we did) - but not 100% sure. @ultraschuppi can you clarify please?

```suggestion
- On the stable channel, the last three supported minor versions are made available for provisioning.
```

Reply:
yep, supported minors it is

Reply:
```suggestion
- On the stable channel, all supported minor versions are made available for provisioning.
```
@MaxTru should we change it to this?

Reply:
Yes - I think this makes sense. Thanks